### DISCUSSION TITLE
NB: Error during install

### DISCUSSION THREAD

**Discussion by revwines** [Homebrew/discussions#3781] [Getting started]
Hi 

Encountered this during the installation of HomeBrew.  I'm really a NB, but it looks like the remote host terminated the download? If so, what's the best way to recover from this?  Start from first step?

HEAD is now at 77311e404 Merge pull request #13994 from Homebrew/set-output-deprecation
==> Tapping homebrew/core
remote: Enumerating objects: 1288532, done.
remote: Counting objects: 100% (553/553), done.
remote: Compressing objects: 100% (248/248), done.
error: RPC failed; curl 56 LibreSSL SSL_read: SSL_ERROR_SYSCALL, errno 54
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed
Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master
bash-3.2$ brew install commandbox
==> Downloading https://ghcr.io/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:1f50bf80583bd436c9542d4fa5ad47df0ef0f0bea22ae710c4f04c42d7560bca
################################################################################################################# 100.0%
==> Pouring portable-ruby-2.6.8_1.el_capitan.bottle.tar.gz
Running `brew update --auto-update`...
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

**Comment by SMillerDev** [comment]
See #666
